### PR TITLE
Added the initial integration test for ListPlans in GroundStationServ…

### PR DESCRIPTION
**Pull request by hoshir**
…ice.

Added the initial integration test class for ListPlans API in GroundStationService. The integration test is expected to be run in test/python directory.

### CODE REVIEW

File: test/python/groundstation/list_plans_test.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+from google.protobuf.timestamp_pb2 import Timestamp
+from stellarstation.api.v1.groundstation import groundstation_pb2
+
+from conn.factory import StubFactory
+
+GS_ID = '27'
+
+
+class TestListPlans(unittest.TestCase):
```
Comment (anuraaga) [inline test/python/groundstation/list_plans_test.py]: Can we use pytest? It has many useful features vs std library.

https://docs.pytest.org/en/latest/

Comment (hoshir) [reply test/python/groundstation/list_plans_test.py]: Thanks, changed to pytest and added very basic tests as the initial version.

File: test/python/groundstation/list_plans_test.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+class TestListPlans(object):
+    def test_list_plans(self, stub_factory):
+        client = stub_factory.get_gs_service_stub()
+
+        from_time = Timestamp(seconds=int(datetime(2018, 12, 1, 0, 0).timestamp()))
+        to_time = Timestamp(seconds=int(datetime(2018, 12, 31, 0, 0).timestamp()))
+
+        request = groundstation_pb2.ListPlansRequest(
+            ground_station_id=GS_ID,
+            aos_after=from_time,
```
Comment (anuraaga) [inline test/python/groundstation/list_plans_test.py]: I think the idiomatic way may be to do `request.aos_after.FromDateTime(datetime(...))` - does it work?

Comment (anuraaga) [reply test/python/groundstation/list_plans_test.py]:
```python
request = groundstation_pb2.ListPlansRequest()
request.ground_station_id = GS_ID
request.aos_after.FromDateTime(...)
request.aos_before.FromDateTime(...)
```

Comment (hoshir) [reply test/python/groundstation/list_plans_test.py]: Thanks, very good to know.

File: test/python/conftest.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Infostellar, Inc.
```
Comment (anuraaga) [inline test/python/conftest.py]: Can you name the folder `integration-tests`?

**Pull request merged**